addons/web/static/tests/fields/relational_fields_tests.js

```diff
@@ -4314,6 +4314,97 @@ QUnit.module('relational_fields', {
         form.destroy();
     });
 
+    QUnit.test('onchange many2many in one2many list editable', function (assert) {
```

Comment:
btw, the name of the test should be more explicit (and if it's complicated to explain in a few words, a comment block on the top of the test is welcome to explain everything in details